src/components/TableRow/styles.css

```diff
@@ -0,0 +1,56 @@
+td:nth-child(1) {
+  width: 100%;
+  display: flex;
+  display: flex;
+  flex-direction: column;
+}
+tr {
+  width: 100%;
+  display: flex;
+  border-bottom: 1px solid rgb(102, 174, 223);
+  margin-bottom: 5px;
+}
+
+tr:hover {
+  background-color: rgb(245, 245, 246);
+}
+
+td p {
```

Comment:
Please do not use `tag` selectors and do not nest any selectors without having really serious reason behind it